### ISSUE TITLE
problem with 1" neutral wire

### ISSUE THREAD

**Issue by dankelley** [dankelley/mooring#40]
---
author: kelley
date: 2021-06-21
output: "reprex::reprex\\_document"
title: satin-urial\_reprex.R
---

``` r
library(mooring)
a <- anchor(depth=110)
W <- wire('1in Neutral Wire',length=100)
w <- wire('1/2in Neutral Wire',length=100)
f <- float("HMB 20")
m <- discretise(mooring(a,w,f))
M <- discretise(mooring(a,W,f))
par(mfrow=c(1,2))
plot(knockdown(m,0.5),which="knockdown",title="thin wire")
plot(knockdown(M,0.5),which="knockdown",title="thick wire (should deform more)")
```

![](https://i.imgur.com/7yhow7d.png)

``` r
w$CD
#> [1] 1.5
W$CD
#> [1] 0
```

<sup>Created on 2021-06-21 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by dankelley** [comment]
The problem is that the CD value is zero.  I'll fix that tomorrow.  I'll also look into whether the radius ratio should be 2 -- perhaps the name is not quite right (this seems very likely) or there is a rounding problem in Dewey's matlab file (also not too unlikely).

**Comment by dankelley** [comment]
Fixed in "main" commit a6a7628fdd8626ae1150084879ee526d5a0d93e2.  I wrote code to detect zero CD in Dewey's file (it had 2 hits).  I replaced them with median values in the category.
